clients/privacy-center/next.config.js

```diff
@@ -1,5 +1,11 @@
-const path = require("path");
-const { version } = require("./package.json");
+const isDebugMode = process.env.FIDES_PRIVACY_CENTER__DEBUG === "true";
```

Comment:
This works very much the same as the `initializeDebugger` does on the client-side, but this is for server-side. Similar to `initializeDebugger` on the client-side, if debug mode is not enabled we just set the function to a noop `() => {}`.

The main difference on the server-side though is that we aren't trying to preserve line numbers/file names because the server doesn't log with those anyway. That being the case, We're able to add a bit of flare to the logs to help distinguish them from other server logs.

Using "Ansi Escape codes" like `\x1b[33m${debugMarker}\x1b[0m` is simply adding color to the `debugMarker`. In this case `\x1b[33m` which is for yellow, and then `\x1b[0m` which sets it back to default color. These escape codes are supported in both NodeJS's console.log as well as browsers.

And, of course, for all intents and purposes, `globalThis` is essentially the equivalent of `window` but in the NodeJS environment.